src/EditorFeatures/Core/IntelliSense/AsyncCompletion/ItemManager.CompletionListUpdater.cs

```diff
@@ -670,8 +682,8 @@ private static bool IsPotentialFilterCharacter(char c)
 
             private ItemSelection UpdateSelectionBasedOnSuggestedDefaults(IReadOnlyList<MatchResult<VSCompletionItem>> items, ItemSelection itemSelection, CancellationToken cancellationToken)
             {
-                // Editor doesn't provide us a list of "default" items.
-                if (_snapshotData.Defaults.IsDefaultOrEmpty)
+                // Editor doesn't provide us a list of "default" items, or we select SuggestionItem (because we are in suggestion mode and have no match in the list)
```

Comment:
might be worthwhile to explain that we represent the SuggestionItem with a negative number.